content/docs/intro/cloud-providers/openstack/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: OpenStack
-meta_desc: This page provides an overview of the OpenStack Provider for Pulumi.
+title: OpenStack Provider
```

Comment:
```suggestion
title: OpenStack
```

@97thFloor How important is it to add provider into the title here? None of the other "providers" have "Provider" in their title so it makes this page look a bit out of place in our nav. If it is a good idea to add provider to the title's we should do it across all the pages and add a field for link text.

Reply:
Most of the pages ranking on page one for this keyword have "provider" in the title, aside from OpenStack themselves, so I'd say keep it if possible. For the other pages I can't say definitely without knowing what keywords they are ranking for, if any. However, since they're all the same format to this page, I feel confident saying it would probably be a good move. From a conversions standpoint, it's a bit more specific, so people looking at the SERP would know that this is a useful tool rather than just an informational page.

Reply:
@cnunciato ☝️ WDYT?

Reply:
I think this is pretty visually noisy, TBH:

![image](https://user-images.githubusercontent.com/274700/76250157-fb5eba00-6201-11ea-819a-d423ce1faa16.png)

So if we could serve the need by having the page title (i.e., the H1) maybe contain the word Provider instead, keeping the navigation clean and scannable, that'd be a better way to go.